src/components/Layout/Sider.js

```diff
@@ -0,0 +1,22 @@
+import { Layout } from 'antd';
+import NavBar from './NavBar';
+
+export function SiderContent() {
+  const { Sider } = Layout;
+  return (
+    <Sider
+      // I would wait on breakpoints and collapsing options until design documents updated to include mobile
```

Comment:
Excellent use of comments to keep the next codebeast informed of the what and why and what if of the situation.